cli/ballerina-cli/src/main/java/io/ballerina/cli/task/CreateExecutableTask.java

```diff
@@ -95,6 +97,19 @@ public void execute(Project project) {
                 start = System.currentTimeMillis();
             }
             jBallerinaBackend.emit(JBallerinaBackend.OutputType.EXEC, executablePath);
+
+            if (enableNativeImage) {
+                String[] command = {"native-image", "-jar", executablePath.toString()};
+                ProcessBuilder builder = new ProcessBuilder();
+                builder.command(command);
+                builder.inheritIO();
+                Process process = builder.start();
+
+                if (process.waitFor() != 0) {
+                    out.println("\t Native-image generation failed");
```

Comment:
Shall we make this more explanatory? e.g. `GraalVM native image generation failed.`

Reply:
Can we print the output of the native-image command?

Reply:
The output of the graalvm native image command shows the progress and in the end I have added a section that says  `GraalVM image generated`